student.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom, :parent_permission
+
+  def initialize(age, name = 'Unknown', classroom = 'Tenth')
```

Comment:
Please include `parent_permission = true` as a parameter for the `Student` class. Although the `parent_permission` is inherited from the `Person` class using `super`.  A user still needs to be able to modify and override the inherited value should a different permission is required.

- There's no need to set the default value for `parent_permission` to `true` within the `super` constructor. This is because the inherited value for `parent_permission` is already `true` since the `Person` class has already set the default value for `parent_permission`.

![image](https://user-images.githubusercontent.com/38088818/231115856-cf97d1e0-f951-4013-a862-6abbbb292a62.png)